StructureDefinition/NHSDigital-Claim.StructureDefinition.xml

```diff
@@ -611,11 +614,6 @@
       <path value="Claim.item.detail.programCode.coding.system" />
       <fixedUri value="https://fhir.nhs.uk/CodeSystem/medicationdispense-endorsement" />
     </element>
-    <element id="Claim.item.detail.programCode:dispensingEndorsement.text">
```

Comment:
Why was this removed? It is useful guidance for dispensers submitting claims on where they need to provide further information. Are you expecting this to be moved elsewhere?

Reply:
Will investigate, could be a forge change